apps/src/templates/rubrics/AiAssessmentFeedback.jsx

```diff
@@ -30,21 +31,38 @@ export default function AiAssessmentFeedback({learningGoalKey}) {
   const receivedTimer = useRef();
   const disappearAfter = 20000;
 
+  const baseUrl = '/learning_goal_ai_evaluation_feedbacks';
+
   const radioAiFeedbackCallback = radioButtonData => {
     setAIFeedback(radioButtonData);
+    if (radioButtonData === thumbsupval) {
+      const bodyData = JSON.stringify({
+        learningGoalAiEvaluationId: aiInfo.id,
+        aiFeedbackApproval: thumbsupval,
+      });
+      HttpClient.post(`${baseUrl}`, bodyData, true, {
+        'Content-Type': 'application/json',
+      }).catch(error => {
+        console.log(error);
```

Comment:
If we're not going to do anything with this, it would be better practice to let the js error propagate, so that it shows up in our error reporting.